### PR TITLE
chore: tag 0.11.1

### PR DESCRIPTION
**Pull request by pjenvey**

### CODE REVIEW

**Review by jrconlin — APPROVED** [review]
wow.

There's a lot in this release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,106 @@
+<a name="0.11.1"></a>
+## 0.11.1 (2022-05-05)
```
Comment (ethowitz) [inline CHANGELOG.md:2]: Should we update this date again?

Comment (pjenvey) [reply CHANGELOG.md:2]: It's really about the date of when this release was tagged so I think it should stay as is

Comment (ethowitz) [reply CHANGELOG.md:2]: Ah, right 🤦🏻

**Comment by ethowitz** [comment]
Do we want this release to include [the cors fix](https://github.com/mozilla-services/syncstorage-rs/commit/221705b7ea74c6dddffd1c5289c53b3ad2cc7522)?

**Comment by pjenvey** [comment]
I'll be tagging a new release from master with the Cors fix among other things. I'm basically abandoning 0.11, this is mostly just to merge the changelog bits :)

**Comment by ethowitz** [comment]
Got it ok sorry for all the noise 😅

**Review by ethowitz — APPROVED** [review]

**Pull request merged**